app/views/partials/team_members_details_table_filter_form.scala.html

```diff
@@ -67,7 +66,9 @@
         @search_and_filter(
             form,
             withFilter = false,
-            searchLabelKey = "group.members.list.filter.label"
+            searchLabelKey = "group.members.list.filter.label",
+            applyButtonKey = Some("filter.apply.singular"), /* Apply filter (not filters) - APB-7104 */
+            clearButtonKey = Some("filter.clear.singular") /* Clear filter (not filters) - APB-7104 */
```

Comment:
these keys are explicitly for the `withFilter = false` case right? could change the search_and_filter partial to use the singular key if false